sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -167,6 +167,7 @@ Set-SPOTenant
  [-PrebuiltModelScope <SyntexFeatureScopeValue>]
  [-PrebuiltModelSelectedSitesList [String[]]]
  [-PrebuiltModelSelectedSitesListOperation <SelectedSitesListOperations>]
+ [-AllowWebPropertyBagUpdateWhenNoScriptIsOn <Boolean>]
```

Comment:
>AllowWebPropertyBagUpdateWhenNoScriptIsOn [](http://example.com/codeflow?start=3&length=41)

putting a wait pending the ongoing discussion for the backend PR

Reply:
@samkabue changed the name of the parameter based on the comments on the backend PR. Please let me know what you think.